packages/elements/src/Header/getHeaderTitle.tsx

```diff
@@ -1,7 +1,7 @@
-import type { HeaderOptions } from '../types';
+import type { StackHeaderOptions } from '@react-navigation/stack/src/types';
```

Comment:
We can't import things from stack here since it's not a dep of elements and user may not have it installed